kettle/README.md

```diff
@@ -100,10 +100,25 @@ gs://<bucket path>: #bucket url
 
 A [postsubmit job](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/test-infra/test-infra-trusted.yaml#L203-L210) runs that pushes Kettle on changes.
 
+# PubSub
+
+Kettle `stream.py` leverages Google Cloud [PubSub] to alert on GCS changes. These events are tied to the `gcs-changes` Topic in the `kubernetes-jenkins` project where Prow job artifacts are collated. Each time an artifact is finalized, a PubSub event is triggered and Kettle collects job information when it sees a resource uploaded called `finished.json` (indicating the build completed).
+
+[Subscriptions] are in Kuberenetes Jenkins Build - PubSub.
+- kettle
+- kettle-staging
```

Comment:
Should the kettle subscription have the same filter as kettle-staging?
```yaml
# gcloud pubsub subscriptions list --project=kubernetes-jenkins
---
ackDeadlineSeconds: 10
expirationPolicy:
  ttl: 2678400s
messageRetentionDuration: 604800s
name: projects/kubernetes-jenkins/subscriptions/kettle
pushConfig: {}
topic: projects/kubernetes-jenkins/topics/gcs-changes
---
ackDeadlineSeconds: 10
expirationPolicy:
  ttl: 2678400s
filter: attributes.eventType = "OBJECT_FINALIZE"
messageRetentionDuration: 604800s
name: projects/kubernetes-jenkins/subscriptions/kettle-staging
pushConfig: {}
topic: projects/kubernetes-jenkins/topics/gcs-changes
```

Reply:
I added the filter as an experiment and wanted to make sure it worked before rebuilding prod. You can't add a filter to an existing subscription actually, so I really wanted to verify before destroying the existing one.

Reply:
I guess my question would be: when we decide to setup subscriptions on other buckets (either adding more, or switching to a community owned bucket), should we use the filter going forward?

Reply:
Ah, yes we would want the filter. Really we would like a suffix filter if that is added as well.